bundles/framework/featuredata/view/SelectByProperties.jsx

```diff
@@ -0,0 +1,266 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { showPopup } from 'oskari-ui/components/window';
+import { Message, Select, TextInput, Button, Checkbox } from 'oskari-ui';
+import { FEATUREDATA_BUNDLE_ID } from './FeatureDataContainer';
+import { FilterTwoTone, MinusCircleTwoTone, PlusCircleTwoTone } from '@ant-design/icons';
+export const FilterTypes = {
+    equals: '=',
+    like: '~=',
+    notEquals: '≠',
+    notLike: '~≠',
+    greaterThan: '>',
+    lessThan: '<',
+    greaterThanOrEqualTo: '≥',
+    lessThanOrEqualTo: '≤'
+};
+
+export const LogicalOperators = {
+    AND: 'AND',
+    OR: 'OR'
+};
+
+const getOptionsFromColumnNames = (columnNames) => {
+    return columnNames?.map(key => {
+        return {
+            label: key,
+            value: key
+        };
+    });
+};
+
+const generateFilterTypeOptions = () => {
+    return Object.keys(FilterTypes).map(key => {
+        const messageKey = 'selectByPropertiesPopup.filterType.' + key;
+        return {
+            label: <Message bundleKey={FEATUREDATA_BUNDLE_ID} messageKey={messageKey}></Message>,
+            value: FilterTypes[key]
+        };
+    });
+};
+
+const generateLogicalOperatorOptions = () => {
+    return Object.keys(LogicalOperators).map(key => {
+        return {
+            label: key,
+            value: LogicalOperators[key]
+        };
+    });
+};
+
+const Funnel = styled('div')`
+    margin-left: auto;
+`;
+
+const StyledSelectMedium = styled(Select)`
+    min-width: 15em;
+    margin-right: .5em;
+`;
+
+const StyledSelectSmall = styled(Select)`
+    min-width: 8em;
+    margin-right: .5em;
+`;
+
+const StyledTextInput = styled(TextInput)`
+    margin-right: .5em;
+`;
+
+const FlexRow = styled('div')`
+    display: flex;
+    padding: 0 .5em .5em .5em;
+    input.validation-error {
+        border 1px solid red;
+    }
+`;
+
+const ButtonsContainer = styled(FlexRow)`
+    justify-content: center;
+`;
+
+const StyledButton = styled(Button)`
+    margin: 0 .5em 0 .5em
+`;
+
+const Buttons = (props) => {
+    const { closePopup, removeFilter, applyFilters, hasErrors } = props;
+    return <ButtonsContainer>
+        <StyledButton onClick={() => { closePopup(); }}>
+            <Message bundleKey={FEATUREDATA_BUNDLE_ID} messageKey={'selectByPropertiesPopup.buttons.cancel'}/>
+        </StyledButton>
```

Comment:
Try replacing this with https://github.com/oskariorg/oskari-frontend/blob/develop/src/react/components/buttons/SecondaryButton.jsx with type `cancel` so we don't need to have the localization in multiple places.

Maybe we could even replace the clearAll with `<SecondaryButton type='reset' />`?